### PR TITLE
feat(docs): add code of conduct

### PR DESCRIPTION
**Pull request by duwenxin99**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+If you see someone violating the code of conduct, you are encouraged to address
+the behavior directly with those involved. Many issues can be resolved quickly
+and easily, and this gives people more control over the outcome of their
+dispute. If you are unable to resolve the matter for any reason, or if the
+behavior is threatening or harassing, report it. We are dedicated to providing
+an environment where participants feel welcome and safe.
+
+Reports should be directed to *googleapis-stewards@google.com*, the
+Project Steward(s) for *Google Cloud Client Libraries*. It is the Project Steward’s duty to
```
Comment (averikitsch) [inline CODE_OF_CONDUCT.md:74]:
```suggestion
Project Steward(s) for this demo. It is the Project Steward’s duty to
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+to engage in behavior that violates the project’s code of conduct.
+
+If you see someone violating the code of conduct, you are encouraged to address
+the behavior directly with those involved. Many issues can be resolved quickly
+and easily, and this gives people more control over the outcome of their
+dispute. If you are unable to resolve the matter for any reason, or if the
+behavior is threatening or harassing, report it. We are dedicated to providing
+an environment where participants feel welcome and safe.
+
+Reports should be directed to *googleapis-stewards@google.com*, the
```
Comment (averikitsch) [inline CODE_OF_CONDUCT.md:73]:
```suggestion
Reports should be directed to *cloud-database-oss-stewards@google.com*, the
```

Comment (duwenxin99) [reply CODE_OF_CONDUCT.md]: Updated project stewards email.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (duwenxin99) [reply CODE_OF_CONDUCT.md]: Updated

**Review by kurtisvg — APPROVED** [review]

**Pull request merged**